.changeset/itchy-clouds-invite.md

```diff
@@ -0,0 +1,14 @@
+---
+"astro": minor
+---
+
+Remove underscore prefix requirement for assets inside content collections. For extensions like `.astro` or `.css`, you can now remove underscores without raising a warning:
```

Comment:
```suggestion
Removes the requirement for non-content files and assets inside content collections to be prefixed with an underscore. For files with extensions like `.astro` or `.css`, you can now remove underscores without seeing a warning in the terminal.
```
For a minor, I'm not sure we need to be quite so terse here, and this gives more words for the people writing blog posts.  :smile:  I would go for "removes the requirement for" to frontload the *nature of the change* for people scanning.

(e.g. "Remove underscore..." could trigger, "Wait, are they removing support for using underscores to prevent building? Do I need to remove my underscores and do something different now?")